format/mp4/boxes.go

```diff
@@ -1367,8 +1367,45 @@ func decodeBox(ctx *decodeContext, d *decode.D, typ string) {
 		d.FieldU32("h_spacing")
 		d.FieldU32("v_spacing")
 	case "uuid":
-		d.FieldRawLen("uuid", 16*8, scalar.RawUUID, uuidNames)
-		d.FieldRawLen("data", d.BitsLeft())
+		uuidBR := d.FieldRawLen("uuid", 16*8, scalar.RawUUID, uuidNames)
+		// TODO: make nicer
+		uuid := d.ReadAllBits(uuidBR)
+
+		switch {
+		case bytes.Equal(uuid, uuidTfxdBytes[:]):
```

Comment:
If you want you could add comment with spec name/version/url etc where the structures come from